src/mono/sample/iOS-NativeAOT/Program.csproj

```diff
@@ -26,12 +31,11 @@
   <ItemGroup>
     <Compile Include="..\iOS\Program.cs" Link="Program.cs" />
     <DirectPInvoke Include="__Internal" />
+    <_ExcludeFromAppDir Include=".dll" />
+    <_ExcludeFromAppDir Include=".json" />
+    <_ExcludeFromAppDir Include="libbootstrapper.o" />
```

Comment:
nit: It would be good to add a condition here `Condition="'$(PublishAotUsingRuntimePack)' == 'true'"` and possibly a comment why this is necessary.